.github/workflows/matrix.yaml

```diff
@@ -11,39 +23,60 @@ on:
 jobs:
   telepresence_matrix:
     strategy:
+      # Each value in the test matrix map is a list of options. This strategy spawns one job for
+      # every element in the cross product of all these lists. The values for that particular cell
+      # of the test matrix are accessed in the job definition via the matrix context below, e.g.:
+      #   ${{ matrix.client_os }}
       matrix:
 
-        # Comment out the full range of dimensions and use just a subset for faster iteration.
-
         client_os: [ubuntu]
         client_arch: [latest]
-#        client_os: [ubuntu-22.04, ubuntu-20.04]
-#        client_arch: [arm, x86]
```

Comment:
Do the example below (`Below is a larger matrix intended to illustrate...`) makes these commented out lines unnecessary?

Reply:
@aosoriodw Which commented out lines are you referring to? (The diff is a bit confusing, it might help to look at the final result.)

Reply:
@rhs Looks great! Approved